ui/src/views/Logout.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <section class="section">
+    <b-message type="is-success">
+      <p>
+        You have been successfully logged out.
+      </p>
+      <p>
+        <router-link :to="{ name: 'Home' }">
+          Back to home
```

Comment:
Following this link will automatically sign you in again, right? If so I'm not sure about keeping this link. Usually my expectation when I sign out and go back to home is to see the home as if I was not signed in.

Another option would be to change the text to something explicit like "Sign in again"?

Reply:
Ok I can change the text, as we don't have a "public" home page (other than the keycloak login page)